### PR TITLE
adds type to legislature metadata

### PR DESCRIPTION
**Pull request by struan**
fixes https://github.com/everypolitician/everypolitician/issues/526

### CODE REVIEW

File: lib/country_data.rb [hunk 1]
```diff
@@ -128,6 +129,10 @@ def popolo
       def statement_count
         json_with_count.last
       end
+
+      def type
+        popolo[:organizations].find { |o| o[:classification] == 'legislature' }[:type] || ''
```
Comment (tmtmtmtm) [inline lib/country_data.rb]: I think it's probably worth removing the duplication with `lname` here by factoring out a `legislature` method that both call.

Comment (tmtmtmtm) [inline lib/country_data.rb]: The `|| ''` worries me a little here. I don't think we'd want an empty value ending up in `countries.json`. This data _should_ be available for every legislature (or at least will be when that country is being rebuilt, and thus its stanza here is being built), so I think I'd raise an exception in any case where this isn't filled in, so we can catch it early.

Comment (struan) [inline lib/country_data.rb]: Ok, there are currently 78 legislatures that don't have that data.

**Comment by tmtmtmtm** [comment]
Thanks @struan — I'll try to make sure every legislature that's still missing these gets rebuilt over the weekend, so that the `everypolitician-ruby` code that consumes this can be ready to go early next week.

**Pull request merged**